webapp/src/app/api/v0/cas/login/login.component.ts

```diff
@@ -45,29 +62,50 @@ export class LoginComponent implements OnInit {
             .replace(/\+/g, "-")
             .replace(/\//g, "_")
 
-    state: State;
-    readonly service: string;
-
-    private ticket: Buffer | undefined;
+    private state: State;
```

Comment:
It seems that javascript supports `state` and `State`. But my brain doesn't. Please chose names that are not only different in the case.